tests/lib/rules/space-infix-ops.js

```diff
@@ -358,6 +373,47 @@ ruleTester.run("space-infix-ops", rule, {
                 column: 6,
                 nodeType: "BinaryExpression"
             }]
+        },
+        {
+            code: "a+b - c",
+            options: { all: "never", "+": "always" },
+            output: "a + b-c",
+            errors: [{
```

Comment:
Hmm. Should this test have multiple errors (one for the plus token lacking spaces, one for the minus token having spaces)?